vowpalwabbit/reductions/cb/cb_explore.cc

```diff
@@ -83,12 +84,10 @@ void predict_or_learn_first(cb_explore& data, single_learner& base, example& ec)
   }
   else
   {
-    uint32_t chosen = ec.pred.multiclass - 1;
+    uint32_t chosen = (data.indexing == 1) ? ec.pred.multiclass - 1 : ec.pred.multiclass;
```

Comment:
Could this just become
```suggestion
    uint32_t chosen = ec.pred.multiclass - data.indexing;
```

Reply:
Removing indexing from this PR